RELEASES.md

```diff
@@ -1,3 +1,68 @@
+Version 1.22.0 (2017-11-23)
+==========================
+
+Language
+--------
+- [`non_snake_case` lint now allows extern no-mangle functions][44966]
+- [Now accepts underscores in unicode escapes][43716]
+- [`#![feature(const_fn)]` is now no longer required for using
```

Comment:
'calling const functions' instead of 'using const functions' might be clearer here